### PR TITLE
Remove SchemaChangeV1

### PR DESCRIPTION
**Pull request by chaoyli**
Alter table logic have been refactored.
So SchemaChangeV1 can be removed.

### CODE REVIEW

File: be/src/agent/agent_server.cpp [hunk 1]
```diff
@@ -328,7 +328,6 @@ void AgentServer::submit_tasks(
             }
             break;
         case TTaskType::ROLLUP:
-        case TTaskType::SCHEMA_CHANGE:
```
Comment (morningman) [inline be/src/agent/agent_server.cpp]: `ROLLUP` can also be removed

File: be/src/agent/task_worker_pool.cpp [hunk 2]
```diff
@@ -46,7 +46,6 @@
 #include "olap/task/engine_checksum_task.h"
 #include "olap/task/engine_clear_alter_task.h"
 #include "olap/task/engine_clone_task.h"
-#include "olap/task/engine_schema_change_task.h"
```
Comment (morningman) [inline be/src/agent/task_worker_pool.cpp]: `engine_clear_alter_task.h` can be removed

File: be/src/agent/task_worker_pool.cpp [hunk 3]
```diff
@@ -564,7 +563,6 @@ void* TaskWorkerPool::_alter_tablet_worker_thread_callback(void* arg_this) {
             TFinishTaskRequest finish_task_request;
             TTaskType::type task_type = agent_task_req.task_type;
             switch (task_type) {
-            case TTaskType::SCHEMA_CHANGE:
```
Comment (morningman) [inline be/src/agent/task_worker_pool.cpp]: ROLLUP can be removed. Check it all round

File: be/src/agent/task_worker_pool.cpp [hunk 4]
```diff
@@ -601,9 +599,6 @@ void TaskWorkerPool::_alter_tablet(
     case TTaskType::ROLLUP:
         process_name = "roll up";
         break;
-    case TTaskType::SCHEMA_CHANGE:
```
Comment (morningman) [inline be/src/agent/task_worker_pool.cpp]: remove ROLLUP, and this switch case is useless

File: be/src/agent/task_worker_pool.cpp [hunk 5]
```diff
@@ -628,11 +623,6 @@ void TaskWorkerPool::_alter_tablet(
             new_schema_hash = agent_task_req.alter_tablet_req_v2.new_schema_hash;
             EngineAlterTabletTask engine_task(agent_task_req.alter_tablet_req_v2, signature, task_type, &error_msgs, process_name);
             sc_status = worker_pool_this->_env->storage_engine()->execute_task(&engine_task);
-        } else {
```
Comment (morningman) [inline be/src/agent/task_worker_pool.cpp]: This `else` and above `if` can be removed together

File: gensrc/thrift/Types.thrift [hunk 6]
```diff
@@ -146,7 +146,7 @@ enum TTaskType {
     CLONE,
     STORAGE_MEDIUM_MIGRATE,
     ROLLUP,
```
Comment (morningman) [inline gensrc/thrift/Types.thrift]: ROLLUP is deprecated too.
And why the comma here is marked as red?

**Review by morningman — CHANGES_REQUESTED** [review]

File: gensrc/thrift/Types.thrift (hunk 6 above)
Comment (chaoyli) [reply gensrc/thrift/Types.thrift]: I think is Types.thrift decode has some settings. I have rewrite the comma for all TTaskType but still is red.

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**